test/e2e/serial/tests/workload_placement_no_nrt.go

```diff
@@ -192,11 +193,8 @@ func updateInfoRefreshPause(fxt *e2efixture.Fixture, newVal nropv1.InfoRefreshPa
 		err = fxt.Client.Get(context.TODO(), nroKey, updatedObj)
 		Expect(err).ToNot(HaveOccurred())
 
-		// TODO replace with updatedObj.Status.MachineConfigPools[0].Config.InfoRefreshPause when OCPBUGS-16058 is resolved.
-		// currently the functionality of the new flag works bt is matter of not reflecting it in resources status. also we
-		// wait for the respective ds to get ready with the new pods so we should be safe
-		if *updatedObj.Spec.NodeGroups[0].Config.InfoRefreshPause != newVal {
-			klog.Warningf("resource status is not updated yet: expected %q found %q", newVal, *updatedObj.Spec.NodeGroups[0].Config.InfoRefreshPause)
+		if *updatedObj.Status.MachineConfigPools[0].Config.InfoRefreshPause != newVal {
```

Comment:
Will the status info present in `updatedObj.Status.MachineConfigPools[]` for the hypershift case as well?

Reply:
addressed, thanks for pointing it out.